pkg/runtime/runtime.go

```diff
@@ -0,0 +1,51 @@
+package runtime
+
+import (
+	"github.com/elotl/itzo/pkg/api"
+	"github.com/elotl/itzo/pkg/logbuf"
+)
+
```

Comment:
@ldx @jrroman pinging you guys here, as this PR isn't ready yet, but those interfaces shouldn't change much. Let me know what you think about the design.

I think next step will be refactoring itzo runtime (I've already started that in this PR) and making sure that behavior is unchanged and tests are passing. Once it's done I can finish podman's implementation of those interfaces (should be pretty straight forward). Any feedback much appreciated;)